### DISCUSSION TITLE
Has Prisma 5.14 significantly better performance than 5.0?

### DISCUSSION THREAD

**Discussion by peakala** [prisma/prisma#24450] [Q&A]
### Question

Hello,
we are currently using Prisma 5.0 in our production environment.
We are discussing whether we might get significant performance improvement by adopting the newest version.

Please, can anyone share his experience/knowledge/known literature about this?

As far as you know, do you think we might face with a lot of refactor?

Thank you,

PS I'm aware these question can sound a bit risky, however any suggestion is very much appreciated.
PPSS Do you know any performance comparator for ORMs, like a website that compares ORMs performances?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @peakala 👋 

Thank you for raising this question.

We would definitely recommend you to upgrade to the latest Prisma Version [5.15.0](https://github.com/prisma/prisma/releases/tag/5.15.0) because we have fixed a lot of bugs and added new functionalities which might be helpful to you. In Prisma [5.14.0](https://github.com/prisma/prisma/releases/tag/5.14.0), we shipped a lot of performance updates for MongoDB. We would recommend you to go through this [release page](https://github.com/prisma/prisma/releases/) to get additional details on what bug fixes and performance improvements have been shipped.

> Do you know any performance comparator for ORMs, like a website that compares ORMs performances?

We would recommend you to go through this [tool](https://db-latency.vercel.app/) for comparing ORM performances with different Database providers.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
